.gitmodules

```diff
@@ -0,0 +1,27 @@
+[submodule "External/amd-appsdk/amd-appsdk-submodule"]
+	path = External/amd-appsdk/amd-appsdk-submodule
+	url = https://github.com/GPUOpen-Tools/common-lib-amd-APPSDK-3.0
+[submodule "External/opencv/opencv-submodule"]
+	path = External/opencv/opencv-submodule
+	url = https://github.com/opencv/opencv.git
+[submodule "External/openexr/openexr-submodule"]
+	path = External/openexr/openexr-submodule
+	url = https://github.com/openexr/openexr.git
+[submodule "External/zlib/zlib-submodule"]
+	path = External/zlib/zlib-submodule
+	url = https://github.com/madler/zlib
+[submodule "External/libpng/libpng-submodule"]
+	path = External/libpng/libpng-submodule
+	url = https://github.com/glennrp/libpng.git
+[submodule "External/libtiff/libtiff-submodule"]
+	path = External/libtiff/libtiff-submodule
+	url = https://github.com/UebelAndre/compressonator-tiff.git
```

Comment:
This was removed but was a dump of the latest version of http://www.libtiff.org/ into a git repo. It was actually unused but would be a nice quality of life thing to build dependencies with it

Reply:
@UebelAndre  Image format support is user-definable via plugins. CMP will maintain a set of defaults either from existing plugins or from those supported by Qt. The lib module has been added to the common folder and will be reviewed for additional use later!.